src/view/background/GraphBackgroundPainter.cpp

```diff
@@ -8,6 +8,21 @@ GraphBackgroundPainter::GraphBackgroundPainter() = default;
 
 GraphBackgroundPainter::~GraphBackgroundPainter() = default;
 
+/**
+ * Set the Graph line color to a lower contrast alternative if a black background is used
+ */
+void GraphBackgroundPainter::updateGraphColor() {
+    unsigned int backgroundColor = this->page->getBackgroundColor();
+
+    // The background color returned by getBackgroundColor is sometimes offset by 0xff000000
+    if (backgroundColor == 0x000000 || backgroundColor == 0xff000000) {
+        this->foregroundColor1 = 0x202020;
+        return;
+    }
+
+    this->foregroundColor1 = 0xBDBDBD;
```

Comment:
This only works for color 0x0. It would be better to check for the distance.
```suggestion
   auto greyscale = [](uint32_t color){
       return ((0xff & color) + (0xff & (color >> 8)) + (0xff & (color >> 16))) / 3;
   };

this->foregroundColor1 = std::abs(int32_t(greyscale(backgroundColor)) - int32_t(greyscale(this->foregroundColor1))) < 30 ? 0x202020 : 0xBDBDBD;
```